client/src/utils/getApplicationStatus.tsx

```diff
@@ -1,16 +1,16 @@
 
-const getApplicationStatus = (status: boolean) => {
+const getApplicationStatus = (status: boolean | number) => {
   switch (status) {
-    case true:
+    case 0:
       return (
```

Comment:
Create constants in utils for `ApplicationStatus.PENDING` and `ApplicationStatus.APPROVED`